pyomo/solvers/tests/mip/test_convert.py

```diff
@@ -118,7 +119,7 @@ def test_nl_lp1(self):
                           "enabled but not available: '%s'" % str(err))
             return
         self.assertEqual(ans[0][0][-15:],"pico_convert.lp")
-        self.assertFileEqualsBaseline(ans[0][0], currdir+"test1_convert.lp")
+        self.assertTrue(cmp(ans[0][0], currdir+"test1_convert.lp"))
```

Comment:
As with above, it would be helpful to include a failure message here (and the other use below).